src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -95,6 +95,15 @@ final class WP_Interactivity_API {
 	 */
 	private $context_stack = null;
 
+	/**
+	 * Representation in array format of the element currently being processed.
+	 *
+	 * This is only available during directive processing, otherwise it is `null`.
+	 *
+	 * @since 6.7.0
+	 * @var array<mixed>|null
+	 */
+	private $current_element = null;
 	/**
```

Comment:
Missing space here

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/7474